projects/ngrx.io/content/examples/store-walkthrough/src/app/models/book.ts

```diff
@@ -0,0 +1,17 @@
+export interface Book {
```

Comment:
There's already a `Book` interface that we should reuse [here](https://github.com/ngrx/platform/blob/master/projects/ngrx.io/content/examples/store-walkthrough/src/app/book-list/books.model.ts). If we reuse that model, we should move it to this file, and remove the old file,

Reply:
Oh, I didn't see it 😄
Done! Thanks for the tip